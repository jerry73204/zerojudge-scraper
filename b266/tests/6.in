5 8 71
4 6 0 0 3 3 4 0
1 5 2 2 6 4 9 8
9 7 4 4 4 1 4 5
6 6 9 0 0 1 1 7
5 5 2 0 3 9 4 0
1 1 0 0 0 0 1 1 1 0 1 1 1 0 0 1 1 0 0 1 0 0 1 1 0 1 1 1 0 0 0 0 1 1 0 1 0 1 1 0 1 1 1 1 1 0 1 0 1 0 1 1 1 0 0 0 0 1 0 1 0 0 1 0 0 1 0 1 0 1 0
