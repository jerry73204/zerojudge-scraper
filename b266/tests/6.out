5 8
5 5 2 0 3 9 4 0
6 6 9 0 0 1 1 7
9 7 4 4 4 1 4 5
1 5 2 2 6 4 9 8
4 6 0 0 3 3 4 0
